api-utils/apiHooks.ts

```diff
@@ -1,6 +1,10 @@
-import { APIRequestContext } from "@playwright/test";
+import { APIRequestContext, APIResponse } from "@playwright/test";
+import { ApiUtils } from "./apiUtils";
 
 export class ApiHooks {
+  // TODO: get last created task id from response
+  private static lastCreatedTaskId: APIResponse;
```

Comment:
Dwie rzeczy:
- na ten moment takie rozwiązanie wydaje się okej, ale może się pojawić problem, kiedy np. będziemy przez API tworzyli wiele tasków - wtedy będziemy mieli dostęp tylko do `id` ostatniego taska
- nazwa pola wskazuje, że to `id`, ale typ wskazuje, że jest to cały `response` z całym ciałem odpowiedzi, nagłówkami itp.

Reply:
Racja, ogarnę to. Już jest to zrobione w taki sposób, że id są brane przez requesty i lastCreatedTaskId nie jest w tym momencie używane.